src/sync/builder.rs

```diff
@@ -380,6 +388,18 @@ impl<K, V, C> CacheBuilder<K, V, C> {
             ..self
         }
     }
+
+    /// Specify whether or not to enable thread pool for housekeeping tasks, such as
+    /// ... `true` to enable and `false` to disable.
+    ///
+    /// The thread pool is enabled by default in current version of Moka but the
+    /// default will be changed to disabled in future version.
```

Comment:
Need to finish writing the doc.

Reply:
Done via https://github.com/moka-rs/moka/pull/165/commits/4041e1fd8914eb16ae5aa8d25dab0207124fd05b.